src/redis-cli.c

```diff
@@ -1643,7 +1643,7 @@ static int parseOptions(int argc, char **argv) {
     }
 
     if (!config.no_auth_warning && config.auth != NULL) {
-        fputs("Warning: Using a password with '-a' or '-u' option on the command"
+        fputs("Warning: Using a password with '-a' or '--pass' option on the command"
```

Comment:
I think `-u` should not be removed here.
`-u` means uri, like this: `redis://user:password@host:port/dbnum`.

By the way:
For `--pass`, it is an alias name for `-a`, whether add it here or not both are ok for me.

Reply:
Got it. Re-added '-u'